src/component/maincomponent.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import Header from './header';
+import Content from './content';
+import Footer from './footer'
+
+
+class Main extends Component{
+    constructor(props){
+        super(props);
+        this.state={
+            FirstName: ['pedro','devon'],
+            LastName: ['sanchez','gonzalez']
+        };
+    }
+
+    render(){
+        return(
+            <div>
+                <Header/>
+                <Content firstname={this.state.FirstName} lastname={this.state.LastName}/>
```

Comment:
convention is camelCasing also here we are passing directly an array this would most likely output a `[object Object]` string so we ideally want to pass the string directly instead and adjust on the child component as mentioned on above comment
```suggestion
                <Content firstName={this.state.firstName[0]} lastName={this.state.lastName[0]} />
```

Reply:
Yes. looks Good.